fxdc/exceptions.py

```
@@ -2,9 +2,7 @@
 class FXDCException(Exception):
     def __init__(self, *args, **kw) -> None:
```

Ruff (0.12.2)

Missing type annotation for `*args`

(ANN002)